movie/src/components/PopupCard/BoxDisplayGenres/BoxDisplayGenres.css

```diff
@@ -0,0 +1,35 @@
+.genreBox {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 300px;
+}
+
+.labelUnClick {
+  border-radius: 10px;
+  font-weight: 700px;
+  font-size: 0.875rem;
+  padding: 5px;
+  border: 1px solid gray;
+}
+
+.labelUnClick:hover {
+  background-color: rgb(23, 144, 144);
+  color: white;
+}
+
+.labelClicked {
```

Comment:
its looks like your `labelUnClick` and `labelClicked` has most of the same CSS properties. So create a base CSS class for it